### DISCUSSION TITLE
Kedro "Hello World" tutorial/example does not work

### DISCUSSION THREAD

**Discussion by Jason-F** [kedro-org/kedro#1516] [Idea]
The kedro hello world example from here: https://kedro.readthedocs.io/en/stable/get_started/hello_kedro.html does not work.

It gives an error about missing 1 required positional argument: 'hook manager'

Is the rest of the kedro documentation also outdated(I heard this example is outdated due to a .py change in the repo) or is there something I'm missing here?

**Comment by noklam** [comment]
https://github.com/kedro-org/kedro/issues/1428

Please find a workaround here, we have a fix for it and will be release next week.

**Reply by Jason-F** [reply]
> 

Awesome. Thanks for letting me know about the fix. Is the spaceflight tutorial fine to use? I need to learn how to use Kedro for work and this error has me worried. Also do you know of a workaround to this issue?

Edit: I found the workaround for this issue with hello world

**Reply by noklam** [reply]
Tutorial will work, running Kedro in a project is more common. The Hello World example introduces a few Kedro concepts but you don't really need to manage these runner, catalog object for a project. It is all handled by the framework.

**Reply by noklam** [reply]
For the record, this should not be a problem anymore since `kedro==0.18.1`
